docs/developer/testing.md

```diff
@@ -0,0 +1,188 @@
+# Testing guidance
+
+When building Web sites and apps on FAST-DNA it's recommended to perform exhaustive testing to instill the confidence that you've met your customers' expectations.
+
+## Quality checklist
+
+1. Test each page for design and content issues
+    1. Test screen sizes from viewport 1 to 6
+    2. Test screen orientations
+       1. Portrait
+       2. Landscape
+    3. Test word length and case
+        1. Test smallest amount of text
+        2. Test largest amount of text
+        3. Case sensitivity
+2. Test for localization (RTL, LTR)
+    1. English
+    2. Arabic
+    3. Russian
+    4. French
+    5. Japanese
+    6. Korean
+3. Test for progressive enhancement
+    1. Test complete browser matrix
+    2. Test for zooming browser from 200% and 25% 
+4. Test each component and pages for accessibility issues
+    1. Test for keyboard accessibility
+    2. Test for Mouse only interaction
+    3. Test for Touch interaction
+    4. Test with High Contrast of OS
+    5. Test without images
+    6. Test for text only
+    7. Test with a screen reader
+    8. Test with VoiceOver
```

Comment:
isn't VoiceOver a screen reader? - that seems duplicate (but more specific) - maybe consider consolidating.

Reply:
It's specific to iOS. I can remove as duplicate.  thx